tests/app/questionnaire/test_value_source_resolver.py

```diff
@@ -59,7 +59,7 @@ def get_calculation_block(
 
 def get_value_source_resolver(
     schema: QuestionnaireSchema = None,
-    data_stores: DataStores = DataStores(),
+    data_stores: DataStores = None,
     location: Union[Location, RelationshipLocation] = Location(
```

Comment:
same with this one, if this function handles the default being None with `data_stores or DataStores()` then it doesn't need to be passed in everywhere

Reply:
Done.